packages/terra-notification-dialog/src/NotificationDialog.scss

```diff
@@ -0,0 +1,126 @@
+:local {
+  .notification-dialog {
+    background: var(--notification-dialog-background, #fff);
+    border-radius: var(--notification-dialog-border-radius, 0.3125rem);
```

Comment:
Let's update to match same value as modal-manager:
`border-radius: var(--notification-dialog-border-radius, 5px);`